### ISSUE TITLE
to use your pretrained models, what should the classe feed into _fasterRNCC should be?

### ISSUE THREAD

**Issue by xuxuzhou** [jwyang/faster-rcnn.pytorch#390]
Cause I don't know the order of MSCOCO classes while you train the model, can you provide a file contain the classes's order?

**Comment by huawei-tomas** [comment]
Check [this](https://github.com/amikelive/coco-labels/blob/master/coco-labels-2014_2017.txt) out. Found [here](https://tech.amikelive.com/node-718/what-object-categories-labels-are-in-coco-dataset/).
